modules/README.md

```diff
@@ -44,14 +44,15 @@ that will be passed to the Puppet run.
 You can test the module without running a pxp-agent. Make sure the module is
 executable and you can run it like:
 
+### Posix
 ```
 $ sudo echo "{\"params\":{\"env\":[],\"flags\":[\"--noop\"]}, \"config\" : {\"puppet_bin\" : \"/opt/puppetlabs/bin/puppet\"}}" | puppet run
```

Comment:
While you're in here, might want to update this one as well.

Should be `| pxp-module-puppet run`, not `| puppet run`